### DISCUSSION TITLE
THE WALLPAPERS

### DISCUSSION THREAD

**Discussion by ThinkyMiner** [prasanthrangan/hyprdots#1246] [General]
Can someone please help me with the source of the wallpapers. like if I can have the exact wallpapers. Pleaseeee!!!!!

**Comment by Gaurav-Gosain** [comment]
https://github.com/prasanthrangan/hyprdots/tree/main/Configs/.config/swww Does this help?

**Reply by Gaurav-Gosain** [reply]
Click through inside the sub-directories.

**Reply by anakinsonone** [reply]
The link no longer works. I guess they have moved the wallpapers somewhere else, I tried looking for them inside all of the sub-directories, but still couldn't find one.

**Comment by kRHYME7** [comment]
https://github.com/kRHYME7/hyde-gallery

See the list of the themes. Themes are maintained on different repo

**Reply by lokesh-krishna** [reply]
Even that repo doesn't have the wallpapers though.

**Reply by kRHYME7** [reply]
> Even that repo doesn't have the wallpapers though.

Cause the repo only list the links of the themes. check ReadME. 
![image](https://github.com/user-attachments/assets/083e1cf6-ec58-4ce4-9c80-266e00196901)

**Comment by ArikRahman** [comment]
Where is the best source for it now? I can only find them through old branches

**Reply by kRHYME7** [reply]
in themes.

**Comment by ArikRahman** [comment]
Thanks, I found it here, changing the branch to choose which wallpaper themes you want: https://github.com/HyDE-Project/hyde-themes

**Comment by ArikRahman** [comment]
Just an update, my solution before doesn't work. It seems like all the files have been compressed. it seems as if the author is doing a port?
